src/components/Profile-Page/index.js

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import { Link } from "gatsby";
+import { ProfileCardWrapper } from "./ProfilePage.style";
+import { FaTwitter, FaGithub, FaLinkedin } from "react-icons/fa";
+import MeshMateIcon from "../../svgs/meshmate-icon.svg";
+import MeshMateIconWhite from "../../svgs/meshmate-icon.svg";
+import Image from "../image";
+import {Col, Container, Row} from "../../reusecore/Layout";
+
+const ProfileCard = (props) =>{
```

Comment:
@vinayaksh42 how do we avoid repeating this const between the Profile-Page and Profile-Card components?

Reply:
I think this is a typo on @vinayaksh42 part. If I am not wrong the contents inside this function are different. If so this should be changed to something like `ProfileInformation` and also this should be sections folder

Reply:
Even, I dont think the usage of const is required. It can be directly accessed.